src/main/resources/l10n/JabRef_en.properties

```diff
@@ -386,6 +386,7 @@ Edit\ entry=Edit entry
 Save\ file=Save file
 Edit\ file\ type=Edit file type
 
+Add\ group=Add group
```

Comment:
There is already `add\ group` in line 57 of the same file. We really don't need localizations that differ only in the casing of the first word. Please remove the old (lower case) localization and make sure that everything still works.

Reply:
The old localization line is used by  org.jabref.gui.groups.UndoableAddOrRemoveGroup.getPresentationName() to name a undoable/redoable action. The problem is, that all actions of this kind are named in lowercase, it would be awkward to change just this name to uppercase.

Reply:
I'm not happy with having the essentially same translation key twice, but I get your point. We can leave this as it is.